price_feeder/tasks.py

```diff
@@ -33,7 +33,7 @@ def pending_queue_is_full(account_index=0):
     last_used_nonce = web3.eth.getTransactionCount(account_address)
 
     # A limit of pending on blockchain
-    if nonce >= last_used_nonce + 2:
+    if nonce >= last_used_nonce + 1:
```

Comment:
:thinking: Why just one? maybe a comment can clarify. Maybe configurable?

Reply:
that is the original value